packages/components/src/theme-type-badge/bundled-badge/index.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { FunctionComponent, useRef, useMemo, useState } from 'react';
```

Comment:
Nit: can you import only the `FunctionComponent` type?
```suggestion
import { type FunctionComponent, useRef, useMemo, useState } from 'react';
```